test/fixtures/color-index.output.css

```diff
@@ -1,5 +1,5 @@
 @media screen and (min-color-index: 0) and (max-color-index: 8) {}
 
-@media screen and (min-color-index: 0) and (max-color-index: 8) {}
+@media screen and (min-color-index:0 ) and (max-color-index: 8) {}
```

Comment:
Is it possible to not output extra spaces here `(min-color-index:0 )`?

Reply:
This is the space that was originally placed after `0` in something like `(0 < color-index < 8)`

I've been considering adding a serializer function to the package based on : https://www.w3.org/TR/cssom-1/#serializing-media-queries

Would this be a good solution?

Or is it preferable to have some heuristics that try to preserve the original code style/format?

Reply:
Also changed a few other bits.
There no longer are any differences in existing tests.